app/src/main/java/org/simple/clinic/crash/SentryCrashReporter.kt

```diff
@@ -23,7 +25,29 @@ class SentryCrashReporter(
 
   override fun init(appContext: Application) {
     Sentry.init(AndroidSentryClientFactory(appContext))
+    keepUserAndFacilityRegistered()
+  }
+
+  @Suppress("CheckResult")
+  private fun keepUserAndFacilityRegistered() {
+    val loggedInUserStream = userSession.loggedInUser()
+        .subscribeOn(io())
+        .filterAndUnwrapJust()
+        .replay()
+        .refCount()
+
+    loggedInUserStream
+        .map { it.uuid }
+        .subscribe({
+          Sentry.getContext().addTag("userUuid", it.toString())
+        }, { report(it) })
```

Comment:
This formatting looks strange. Let's keep the arguments on separate lines?

```kotlin
.subscribe(
    { Sentry.getContext().addTag("userUuid", it.toString()) },
    { report(it) })
```

Reply:
Done